core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -143,22 +175,98 @@ var TopicEditorPage = function() {
     });
   };
 
+  this.selectSkillWithDescriptionFromDropDown = function(skillDescription) {
+    waitFor.elementToBeClickable(
+      selectSkillDropdown, 'Skills dropdown taking too long to appear.');
+    selectSkillDropdown.click();
+    waitFor.elementToBeClickable(
+      element(by.css('option[label="' + skillDescription + '"]'))
+      , 'Skill from dropdown taking too long to be clickable'
+    );
+    element(by.css('option[label="' + skillDescription + '"]')).click();
+  };
+
+  this.openQuestionEditor = function(index) {
```

Comment:
We don't need async-await here because we have used `.then()` here.
In general, I have tried using .then() wherever possible until the migration is completed.
Async-await is only used in places where using `.then()` chains can make the code very unreadable and buggy.
Although we can use async-await if needed.